docs/edex/new-grid.md

```diff
@@ -15,105 +17,52 @@ If the parameters in the grib file haven't been previously specified, another ch
     
-        cp cpti.grib2 /awips2/data_store/ingest/
+    ...
     
-    Remember that the data distribution file (`/awips2/edex/data/utility/common_static/base/distribution/grib.xml`) will match filenames which have the `*.grib*` extension.
+!!! warning "This step may fail if the parameter is not yet defined.  The error will look like:"
 
-2. Confirm that the grib file decodes in the grib log file:
-    
-        edex log grib
+<pre>
+INFO  2020-07-20 20:34:17,710 2565 [GribPersist-1] GridDao: EDEX - Discarding record due to missing or unknown parameter mapping: /grid/2018-05-02_00:01:44.0_(0)/GribModel:161:0:97/null/null/403/Missing/FH/500.0/-999999.0
+INFO  2020-07-20 20:34:17,710 2566 [GribPersist-1] Ingest: EDEX: Ingest - grib2:: /awips2/data_store/ingest/CPTI_00.50_20180502-000144.grib2 processed in: 2.3550 (sec)
+INFO  2020-07-20 20:34:17,827 2567 [Ingest.GribDecode-6] grib: EDEX - No parameter information for center[161], subcenter[0], tableName[4.2.209.3], parameter value[61]
+</pre>
     
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/cpti.grib2 processed in: 0.1200 (sec) Latency: 21.8080 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/cpti.grib2 processed in: 0.1180 (sec) Latency: 21.8140 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/cpti.grib2 processed in: 0.4230 (sec) Latency: 21.8360 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/cpti.grib2 processed in: 0.2240 (sec) Latency: 21.9140 (sec)
-        
-        ...
-        
-    **Note:** This step will likely fail, because the parameter is not yet defined.  The error will look like:
+In order to successfully ingest the example file, you must [**define the appropriate table**](#adding-a-table).
 
-        INFO  2020-07-20 20:34:17,710 2565 [GribPersist-1] GridDao: EDEX - Discarding record due to missing or unknown parameter mapping: /grid/2018-05-02_00:01:44.0_(0)/GribModel:161:0:97/null/null/403/Missing/FH/500.0/-999999.0
-        INFO  2020-07-20 20:34:17,710 2566 [GribPersist-1] Ingest: EDEX: Ingest - grib2:: /awips2/data_store/ingest/CPTI_00.50_20180502-000144.grib2 processed in: 2.3550 (sec)
-        INFO  2020-07-20 20:34:17,827 2567 [Ingest.GribDecode-6] grib: EDEX - No parameter information for center[161], subcenter[0], tableName[4.2.209.3], parameter value[61]
-        
-    In order to successfully ingest the example file, [**define the appropriate table**](#grib2-products_4).
+### Check HDF5 Data
 
-3. Check that the hdf5 data directory exists for our unnamed grid
+Check that the hdf5 data directory exists for our unnamed grid
 
         ls -latr /awips2/edex/data/hdf5/grid/GribModel:161:0:97
 
-    Though the grib file has been decoded, it has been given a generic name with its center, subcenter, and process IDs (161, 0, 97, respectively). 
+    Though the grib file has been decoded, it has been given a generic name with its **center, subcenter, and process IDs** (161, 0, 97, respectively). 
```

Comment:
check the formatting on this, it's currently in the "code block" 
![image](https://user-images.githubusercontent.com/65614396/162507657-a92b33cf-4197-4cd2-9360-b1b503b61ae0.png)